SpringProject/src/main/java/com/gradproject2019/conferences/service/ConferenceServiceImpl.java

```diff
@@ -90,9 +96,17 @@ private void checkConferenceExists(Long conferenceId) {
 
     private void checkUserAuthorised(UUID token) {
         try {
-            authServiceImpl.checkTokenExists(token);
+            authService.checkTokenExists(token);
         } catch (TokenNotFoundException e) {
             throw new UserUnauthorisedException();
         }
     }
+
+    private void checkIfUserIsInterested(Long conferenceId) {
```

Comment:
What is this method doing at the moment?

Reply:
Nothing, it was intended as a bare bones for checking if a user is interested in a conference before we delete it however this is more relevant to a different ticket.

Reply:
It should probably be added in the other PR then, else you'll have a bunch of fluff in your code base :)